src/components/Modal.js

```diff
@@ -1,11 +1,16 @@
 /* eslint-disable react/prop-types */
 import styled from "styled-components";
+import { COLORS } from "../assets/constants";
 
 const Wrapper = styled.div`
   display: ${props => (props.nodeId ? "block" : "none")};
-  width: 150px;
-  padding: 3px;
-  background-color: #ffffff;
+  width: 180px;
+  padding: 8px;
```

Comment:
패딩값도 `SIZE` 에 상수로 선언되어있어서 사용하면 좋을 것 같아요
```suggestion
  padding: ${SIZE.PADDING}px;
```